loris/resolver.py

```diff
@@ -61,13 +65,40 @@ def resolve(self, ident):
             ident (str):
                 The identifier for the image.
         Returns:
-            (str, str): (fp, format)
+            ImageInfo: Partially constructed ImageInfo object 
```

Comment:
Hmm, I take that back. _get_info just has a little test between resolve() and from_image_file() that we should be able to move to the constructor. Will try it...